libparsec/crates/client/tests/unit/invite/bootstrap_organization.rs

```diff
@@ -8,16 +8,26 @@ use libparsec_types::prelude::*;
 
 use crate::{
     bootstrap_organization, test_organization_bootstrap_finalize_ctx_factory,
-    BootstrapOrganizationError, Client, ClientConfig, EventBus, WorkspaceStorageCacheSize,
+    BootstrapOrganizationError, Client, ClientConfig, EventBus, MountpointMountStrategy,
+    WorkspaceStorageCacheSize,
 };
 
+// should be the same as bootstrap token defined in server/parsec/backend.py L91 as TEST_BOOTSTRAP_TOKEN
+const TEST_BOOTSTRAP_TOKEN_STR: &str = "672bc6ba9c43455da28344e975dc72b7";
```

Comment:
Using [`LazyLock`](https://doc.rust-lang.org/stable/std/sync/struct.LazyLock.html) would be a good fit here but we are on 1.75 :cry:

Reply:
Yeah but maybe soon we'll bump the version ^^ By the way, I wonder if it would be a good idea to maybe move that to where the testbed is defined, and use it both here and in the server's backend, to avoid defining it twice.

Reply:
I would say open an issue

Reply:
#8447